website/docs/reference/api/legacy/unleash/admin/metrics.md

```diff
@@ -8,8 +8,13 @@ title: /api/admin/metrics
 # This document describes the metrics endpoint for admin ui
 
 ### Seen-toggles {#seen-toggles}
+:::caution Deprecation notice
 
-`GET http://unleash.host.com/api/admin/seen-toggles`
+This endpoint has been deprecated
```

Comment:
What should they use instead?

```suggestion
This endpoint has been deprecated. Use the [what?] endpoint instead.
```

Reply:
So this functionality is just going away? In that case, we should say that instead.

Reply:
At least it's not listed in the new docs: https://docs.getunleash.io/reference/api/unleash/metrics there are some new endpoints but I don't think it'd be easy for customers to use them. But we can say that this will be removed in v6